statistics.md

```diff
@@ -331,7 +331,33 @@ When the ratio of the number of modified rows to the total number of rows of `tb
 >
 > Currently, the automatic update does not record the configuration items input at manual `ANALYZE`. Therefore, when you use the `WITH` syntax to control the collecting behavior of `ANALYZE`, you need to manually set scheduled tasks to collect statistics.
 
-Before v5.0, when you execute a query, TiDB collects feedback with `feedback-probability` and updates the histogram and Count-Min Sketch based on the feedback. **Since v5.0, this feature is disabled by default, and it is not recommended to enable this feature.**
+Before TiDB v5.0, when you execute a query, TiDB collects feedback with `feedback-probability` and updates the histogram and Count-Min Sketch based on the feedback. **Since v5.0, this feature is disabled by default, and it is not recommended to enable this feature.**
+
+Since TiDB v6.0, TiDB supports using the `KILL` statement to terminate an `ANALYZE` task running in the background. If you find that an `ANALYZE` task running in the background consumes a lot of resources and affects your application, you can terminate the `ANALYZE` task by taking the following steps:
+
+1. Execute the following SQL statement to get the address and task `ID` of the TiDB instance that is executing the background `ANALYZE` task:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    SELECT ci.instance as instance, cp.id as id FROM information_schema.cluster_info ci, information_schema.cluster_processlist cp WHERE ci.status_address = cp.instance and ci.type = 'tidb' and cp.info like 'analyze table %' and cp.user = '' and cp.host = '';
+    ```
+
+    If there is no result, it indicates that no `ANALYZE` task is executing in the background.
```

Comment:
```suggestion
    If there is no result, it indicates that no `ANALYZE` task is running in the background.
```
It seems that _execute_ is intransitive verb?

Reply:
No, "execute" is transitive. See https://www.merriam-webster.com/dictionary/execute. 
But your suggestion looks better. 👍